playbooks/libvirt/openshift-cluster/templates/domain.xml

```diff
@@ -9,8 +9,8 @@
       <ansible:tag>sub-host-type-{{ g_sub_host_type }}</ansible:tag>
     </ansible:tags>
   </metadata>
-  <currentMemory unit='GiB'>1</currentMemory>
-  <vcpu placement='static'>2</vcpu>
+  <currentMemory unit='GiB'>{{ os_libvirt_instance_memory }}</currentMemory>
```

Comment:
Is setting `currentMemory` what you want given the fixed `<memory unit='GiB'>1</memory>` above?
IIUC currentMemory = memory - balloon, it can only be smaller.
https://libvirt.org/formatdomain.html#elementsMemoryAllocation

I've now tried with `--option libvirt_instance_memory=1.5` and got:
`msg: XML error: Invalid value '1.5' for element or attribute './currentMemory[1]'`
Perhaps it's better to switch to MiB units for better granularity?

I've run with `=2` and it created the cluster successfully but all machines have "Current allocation" = "Maximum allocation" = 1024MiB, so apparently it was clamped to not exceed `memory`.